Makefile: Disable parallel make for "make test" because integration t…

michael-berlin opened the pull request:
…ests would race with each other for the same ports.

@enisoc @alainjobart

enisoc commented:
It looks like technically this will disable parallelism for all targets in this Makefile:

https://www.gnu.org/software/make/manual/html_node/Special-Targets.html

LGTM though.

michael-berlin commented:
Thanks for spotting this. You're right :-(

I've added a comment to clarify our intentions and removed "test" as prerequisite because it doesn't get considered anyway.

enisoc commented:
LGTM
